deployment/terraform/dataspace/main.tf

```diff
@@ -86,7 +86,7 @@ resource "azurerm_container_group" "registration-service" {
 
     environment_variables = {
       EDC_CONNECTOR_NAME      = local.connector_name
-      JWT_AUDIENCE            = "${local.registration_service_url}${local.registration_service_path_prefix}"
+      JWT_AUDIENCE            = local.registration_service_url
```

Comment:
we also need to modify it for local setup ? https://github.com/agera-edc/MinimumViableDataspace/blob/3ec52c9e798f76b63008ea92ba5799220e2fde07/system-tests/docker-compose.yml#L146

Reply:
No update needed, the URL still has /authority in it. We just assemble it differently in TF